packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-testing.md

```diff
@@ -2,104 +2,110 @@
 
-- **Unrelated notifications**: Notifications from the operating system outside of the web app can interrupt a user while interacting with a page (for example, "You received a new chat message."). Consider this possibility when designing and developing for screen readers, and keep any notification messaging concise to limit the interruption.
+<Checkbox label={<span>Keyboard navigation via the <kbd>tab</kbd> key can discover all information.</span>} description={<span>If content should be hidden from other assistive technology like a screen reader, <span className="ws-code">aria-hidden="true"</span> should be passed instead.</span>} id="general-criteria-4" />
 
+<Checkbox label="UI elements are understandable and usable." body={<><Checkbox label="When you navigate to an element by keyboard or other assistive technology, you can easily understand and use the item." id="general-criteria-5a" /> <Checkbox label="The flow of information makes sense when navigating." description="Assistive technology (like screen readers) navigate the page in DOM order. If you use CSS to visually reposition elements, they may be announced in a nonsensical sequence. If you need something to appear earlier in the page, try to physically move it earlier in the DOM instead.
+" id="general-criteria-5b" /></>} id="general-criteria-5" /> <br/>
 
-<br />
+### Accessibility points
 
-### Checklist for testing
-These are some of the main areas we check for on the PatternFly team. We recommend that you check for these same areas in your applications, too. 
+<Checkbox label={<span> <b> Structure: </b> The visual information architecture maps to the various rotor menus that exist by default.</span>} body={<><Checkbox label="Rotor has descriptive and concise headings, landmarks, links, form controls, tables, and other elements." id="accessibility-points-7a" />
+<Checkbox label="Heading levels convey structure/content and do not skip levels." description="A common practice is to use a single h1 for the primary headline or logo on a page, h2s for designating major sections, and h3s for supporting sections." id="accessibility-points-7b" />
+<Checkbox label="Landmarks" id="accessibility-points-7c" />
+<Checkbox label="Links" id="accessibility-points-7d" />
+<Checkbox label="Form controls" id="accessibility-points-7e" />
+<Checkbox label="Tables" id="accessibility-points-7f" /></>}  id="accessibility-points-7" />
+<br/>
+<Checkbox label={<span> <b> Labels: </b> </span>} body={<><Checkbox label=<span><b>Link</b> labels  are descriptive, informative, and unique (unless they have the same URL).</span> id="accessibility-points-8a"/> 
 
-**General questions**
+  <Checkbox label={<span><b>Buttons and form controls:</b></span>} description={ <span>
+    <li> All form controls have clear and descriptive labels. </li>
+    <li> Indicate that a button has the expandable control. </li>
```

Comment:
I think there should be some clarification on this one that this line is only applicable to expandable buttons, not all buttons.

Reply:
Could be worth explicitly mentioning `aria-expanded` and how it should always have a boolean value if the button is intended to be expandable